packages/graphql/src/translate/create-connect-and-params.ts

```diff
@@ -147,9 +154,12 @@ function createConnectAndParams({
                     chainStr: `${nodeName}_on_${relatedNode.name}`,
                     recursing: true,
                 });
-                if (onTypeNodeWhereAndParams[0]) {
-                    whereStrs.push(onTypeNodeWhereAndParams[0]);
-                    params = { ...params, ...onTypeNodeWhereAndParams[1] };
+                if (onTypeNodeWhereCypher) {
+                    whereStrs.push(onTypeNodeWhereCypher);
+                    params = { ...params, ...onTypeNodeWhereParams };
+                    if (preComputedSubqueries) {
+                        subquery.push(preComputedSubqueries);
```

Comment:
Should `aggregateWhere` not be set to true here too?

```suggestion
                        subquery.push(preComputedSubqueries);
                        aggregateWhere = true;
```